scrapy/utils/reqser.py

```diff
@@ -47,7 +50,11 @@ def request_from_dict(d, spider=None):
     eb = d['errback']
     if eb and spider:
         eb = _get_method(spider, eb)
-    return Request(
+    try:
```

Comment:
I can make it fail, I was just following (what I thought was) the spirit of your [comment](https://github.com/scrapy/scrapy/pull/2510#discussion_r97566649) @kmike.
It would also be a bit backward incompatible, currently it wouldn't fail but yield a regular `Request` object.